src/api/resolvers/types/mutations/payloads/project/GraphQLRemoveComponentFromProjectInput.ts

```diff
@@ -0,0 +1,25 @@
+import { GraphQLObjectTypeConfig, GraphQLObjectType, GraphQLString, GraphQLNonNull } from "graphql";
```

Comment:
This file name is also wrong

Reply:
fixed with fd7e8df8eefe9e3f7d97ed076de53856563cca0a